lib/fastlane/plugin/wpmreleasetoolkit/helper/configure_helper.rb

```diff
@@ -225,6 +225,10 @@ def self.add_file(params)
 
         UI.user_error!('You must pass a `destination` to `add_file`') unless params[:destination]
```

Comment:
I was tempted to refactor this method to use keyword arguments instead of an hash with parameters. I decided to save it for a later refactor.

cc @AliSoftware in case that's something that's already on your TODO list.